QtCollider/widgets/QcLevelIndicator.h

```diff
@@ -61,6 +77,8 @@ private Q_SLOTS:
   bool _drawPeak;
   int _ticks;
   int _majorTicks;
+	float _stepWidth;
+	int _style;
```

Comment:
Good call, I'll push that change, along with a symbol<>enum conversion on the SC side, following the pattern of other QT stuff.